src/uikit.cs

```diff
@@ -21315,4 +21340,103 @@ interface UITableViewDiffableDataSource<SectionIdentifierType,ItemIdentifierType
+		[Field ("UIActivityItemsConfigurationMetadataKeyTitle")]
+		NSString Title { get; }
+
+		[Field ("UIActivityItemsConfigurationMetadataKeyMessageBody")]
+		NSString MessageBody { get; }
+	}
+
+	delegate NSObject UIActivityItemsConfigurationMetadataProviderHandler (NSString activityItemsConfigurationMetadataKey);
+	delegate NSObject UIActivityItemsConfigurationPerItemMetadataProviderHandler (nint index, NSString activityItemsConfigurationMetadataKey);
+	delegate NSObject UIActivityItemsConfigurationPreviewProviderHandler (nint index, NSString activityItemsConfigurationPreviewIntent, CGSize suggestedSize);
+
+	[NoWatch, NoTV, iOS (13,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface UIActivityItemsConfiguration : UIActivityItemsConfigurationReading {
+
+		[NullAllowed, Export ("localObject", ArgumentSemantic.Strong)]
+		NSObject LocalObject { get; set; }
+
+		[Export ("supportedInteractions", ArgumentSemantic.Copy)]
+		NSString [] WeakSupportedInteractions { get; set; }
+
+		[Advice ("Uses 'UIActivityItemsConfigurationMetadataKey' constants in the function handler.")]
+		[NullAllowed, Export ("metadataProvider", ArgumentSemantic.Strong)]
+		UIActivityItemsConfigurationMetadataProviderHandler MetadataProvider { get; set; }
+
+		[Advice ("Uses 'UIActivityItemsConfigurationMetadataKey' constants in the function handler.")]
+		[NullAllowed, Export ("perItemMetadataProvider", ArgumentSemantic.Strong)]
+		UIActivityItemsConfigurationPerItemMetadataProviderHandler PerItemMetadataProvider { get; set; }
+
+		[Advice ("Uses 'UIActivityItemsConfigurationPreviewIntent' enum constants in the function handler.")]
+		[NullAllowed, Export ("previewProvider", ArgumentSemantic.Strong)]
+		UIActivityItemsConfigurationPreviewProviderHandler PreviewProvider { get; set; }
+
+		[NullAllowed, Export ("applicationActivitiesProvider", ArgumentSemantic.Strong)]
+		Func<UIActivity []> ApplicationActivitiesProvider { get; set; }
+
+		[Static]
+		[Export ("activityItemsConfigurationWithObjects:")]
+		UIActivityItemsConfiguration Create (INSItemProviderWriting [] objects);
+
+		[Static]
+		[Export ("activityItemsConfigurationWithItemProviders:")]
+		UIActivityItemsConfiguration Create (NSItemProvider [] itemProviders);
+
+		[Export ("initWithObjects:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (INSItemProviderWriting[] objects);
+
+		[Export ("initWithItemProviders:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (NSItemProvider[] itemProviders);
+	}
+
+	interface IUIActivityItemsConfigurationReading { }
+
+	[NoWatch, NoTV, iOS (13,0)]
+	[Protocol]
+	interface UIActivityItemsConfigurationReading {
+
+		[Abstract]
+		[Export ("itemProvidersForActivityItemsConfiguration", ArgumentSemantic.Copy)]
+		NSItemProvider [] ItemProvidersForActivityItemsConfiguration { get; }
+
+		[Export ("activityItemsConfigurationSupportsInteraction:")]
+		bool GetActivityItemsConfigurationSupportsInteraction (NSString activityItemsConfigurationInteraction);
```

Comment:
Do we have an enum for this and then expose the string one for advance users?

Reply:
You can't because this is a protocol :)